index.css

```diff
@@ -0,0 +1,485 @@
+@import url("https://fonts.googleapis.com/css2?family=Nanum+Gothic:wght@400;700;800&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Nanum Gothic', verdana, sans-serif;
```

Comment:
- [ ] I kindly suggest that you make use of cocogoose or lato for your fonts as those are the fonts used in the template design.

Find cocogoose [here](https://www.dafont.com/cocogoose.font):